src/main/java/com/wavefront/opentracing/WavefrontTracer.java

```diff
@@ -420,6 +421,19 @@ public Builder excludeJvmMetrics() {
       return this;
     }
 
+    /**
+     * Register custom propogator to support various formats
```

Comment:
nitpick: blank line after this comment and add a period at the end of the sentence.